services/venice-server/src/main/java/com/linkedin/venice/listener/StorageReadRequestsHandler.java

```diff
@@ -660,41 +654,20 @@ private ComputeResponseRecordV1 computeResult(
       ByteBuffer reuseRawValue,
       VeniceCompressor compressor) {
 
-    switch (rocksDBComputeAccessMode) {
-      case SINGLE_GET:
-        reuseValueRecord = GenericRecordChunkingAdapter.INSTANCE.get(
```

Comment:
I assume this API is still used from elsewhere? Or can we remove it?

Reply:
Nope, its getting used in  `private ByteBuffer getValueBytesForKey` in ActiveActiveStoreIngestionTask.java. Guess we could add reusable object in that file.

Reply:
Look carefully if you want to try that. The BB coming out of RocksDB should be used on the spot and we should not hang on to it (e.g. pass it into a buffer for another thread to pick up) otherwise reuse will cause corruption.

Reply:
Venice writers are sequential for same partition and for DCR, if old value wins it should not linger as the write operation is ignored.  Need to check what happens for per field timestamp updates.

Reply:
I see you added reuse in the AASIT, which is great!